src/internal/tester/tconfig/config.go

```diff
@@ -148,6 +152,18 @@ func ReadTestConfig() (map[string]string, error) {
 		os.Getenv(EnvCorsoM365TestSiteID),
 		vpr.GetString(TestCfgSiteID),
 		"10rqc2.sharepoint.com,4892edf5-2ebf-46be-a6e5-a40b2cbf1c1a,38ab6d06-fc82-4417-af93-22d8733c22be")
+	fallbackTo(
+		testEnv,
+		TestCfgTeamID,
+		os.Getenv(EnvCorsoM365TestTeamID),
+		vpr.GetString(TestCfgTeamID),
+		"d288d6bc-4595-4ff5-87a1-6e7fd750aa42")
```

Comment:
any particular reason to have a separate group and team?

Either way, for the team, can we please use `6f24b40d-b13d-4752-980f-f5fb9fba7aa0` (CorsoCITeam).